src/components/VisaProgressBar.tsx

```diff
@@ -13,26 +8,16 @@ export function VisaProgressBar({
   config: FormConfig
   qualifications: Qualifications
 }) {
-  const totalPrompts = Object.values(config.sections).reduce(
+  const totalPrompts = Object.values(config.order).reduce(
     (accumulator, value) => accumulator + value.length,
```

Comment:
Here you will still need to access the actual prompts from the sections, as this will just get the length of the section name.

```diff
-    (accumulator, value) => accumulator + value.length,
+    (accumulator, value) => accumulator + config.sections[value].length,
```